docs(database upgrade): add a note explaining upgrade to minor version before accessing major (#2084)
Scalingo/documentation: src/_posts/databases/elasticsearch/2000-01-01-start.md

restarted with the new database version. When this operation finishes, your application will be
restarted.

{% note %}
Your database needs to be upgraded to the latest minor version before having access to the next major version.

> Per example, your version is 2.3.X and you want to upgrade to 3.1.X. If there is a 2.5.X version, you need to upgrade it to the 2.5.X first.

Author: Thanks !

> Per example, your version is 2.3.X and you want to upgrade to 3.1.X. If there is a 2.5.X version, you need to upgrade it to the 2.5.X first.

Author: ```suggestion
For instance, your version is 2.3.X and you want to upgrade to 3.1.X. If there is a 2.5.X version, you need to upgrade it to the 2.5.X first.
```
